src/main/java/uk/gov/hmcts/reform/unspec/config/scheduler/JobFactory.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.unspec.config.scheduler;
+
+import lombok.RequiredArgsConstructor;
+import org.quartz.spi.TriggerFiredBundle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.scheduling.quartz.SpringBeanJobFactory;
+import org.springframework.stereotype.Component;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Enables DI in classes bootstrapped by quartz.
+ */
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
@Autowired on constructor is only required when we have multiple constructors in our spring components. 
```suggestion
@RequiredArgsConstructor
```